### ISSUE TITLE
Add description to available content

### ISSUE THREAD

**Issue by RitikPatni** [RitikPatni/Front-End-Web-Development-Resources#83]
To add a description to links that are mentioned but have no meaningful description so far

**Comment by ddhira123** [comment]
I have added some resources links here: #92

**Comment by 5hantanu** [comment]
Can I contribute?

**Comment by RitikPatni** [comment]
@5hantanu Yeah sure

**Comment by SamP923** [comment]
Could I also contribute?

**Comment by RitikPatni** [comment]
@SamP923 Yeah sure if you've any other suggestions that could make this list better and more organized feel free to share them

**Comment by utkarsh-shrivastav77** [comment]
I want to contribute in your project

**Comment by azimov01** [comment]
Hi dear!
Which project?

вс, 27 мар. 2022 г., 10:08 Utkarsh Shrivastav ***@***.***>:

**Comment by utkarsh-shrivastav77** [comment]
can you suggest

**Comment by azimov01** [comment]
Yes of course, but what project are you asking about?


вс, 27 мар. 2022 г., 12:20 Utkarsh Shrivastav ***@***.***>:

**Comment by utkarsh-shrivastav77** [comment]
[RitikPatni](https://github.com/RitikPatni) this one

**Comment by RitikPatni** [comment]
@utkarsh-shrivastav77 Yeah, sure you can contribute to this. You can help me with adding updated links for different categories here, or you can add descriptions to existing content if it's not there, or you can raise PR with interesting new stuff that you find

**Comment by ritikaaaa177** [comment]
@RitikPatni I want to contribute to this issue

**Comment by RitikPatni** [comment]
@ritikaaaa177 Feel free to do so. Please raise a PR with any suggestions that you may have, I'll review and merge it.

**Comment by ritikaaaa177** [comment]
okay @RitikPatni

**Comment by himanshivish1999** [comment]
Can I contribute?

**Comment by RitikPatni** [comment]
Yes sure please go ahead @himanshivish1999

**Comment by nonye94** [comment]
Hi, are you still looking for help.
